BuildingBlocks/src/DevelopmentKit.Identity/ValueObjects/IdentityAddress.cs

```diff
@@ -116,5 +124,8 @@ public override object ConvertFrom(ITypeDescriptorContext? context, CultureInfo?
         return ParseUnsafe(stringValue);
     }
 
+    [GeneratedRegex(@"^(?<addressWithoutChecksum>did\:e\:(?<instanceUrl>(?:[a-z0-9]+\.)+[a-z]{2,})\:dids\:(?<identitySpecificPart>[0-9abcdef]{20}))(?<checksum>[0-9abcdef]{2})$")]
```

Comment:
Use the `CHECKSUM_LENGTH`

Reply:
I had already tried that, but failed. I'm unsure that will be feasible:
![image](https://github.com/nmshd/backbone/assets/2041298/a1fdd67d-c284-4316-b24b-762cfede9e78)